modules/mobile/spec/request/user_request_spec.rb

```diff
@@ -28,6 +28,7 @@
     end
 
     before do
+      Flipper.enable_actor(:mobile_v1_lighthouse_facilities, user)
       Flipper.enable('va_online_scheduling')
```

Comment:
@Tonksthebear Is this something that other specs need as well or should I update it to match the `enable_actor` of the other flipper and disable it after the spec?

Reply:
From what I understood of yesterday's discussion, we should be able to just use `enable` as it should reset between each spec. If you see otherwise though, then it's not behaving the way it should